single.php

```diff
@@ -32,7 +32,7 @@
 $page_terms_data                = get_the_terms( $post, 'p4-page-type' );
 $context['background_image']    = $page_meta_data['p4_background_image_override'][0] ?? '';
 $take_action_page               = $page_meta_data['p4_take_action_page'][0] ?? '';
-$context['page_type']           = $page_terms_data[0]->name ?? '';
+$context['page_type']           = 'Post Page';
 $context['page_term_id']        = $page_terms_data[0]->term_id ?? '';
 $context['page_category']       = $category->name ?? __( 'Post page', 'planet4-master-theme' );
```

Comment:
As we are using new `page_type` var for datalayer, so we don't need `page_category` var anymore. right?

Reply:
Changed it back to page_category to avoid name collision.